### PR TITLE
Chat front

### PR DESCRIPTION
**Pull request by se-lee**
- css adjustments - input width is fixed
- some name change on variables for more clarification
- added unmute to the clickable menu (a function to be added)
- updates on member list when an user is blocked

### CODE REVIEW

**Review by Blodispic — APPROVED** [review]
Everything looks good ! :+1:

File: front/src/components/Chat/clickableMenu.tsx [hunk 1]
```diff
@@ -22,6 +22,10 @@ export default function ClickableMenu(props: { user: IUser, chan: IChannel, page
         socket.emit('GiveAdmin', { chanid: props.chan.id, userid: props.user.id });
     }
 
+    const handleUnmute = () => {
+        // socket.emit('Unmute', { chanid: props.chan.id, userid: props.user.id });
+    }
+
```
Comment (Blodispic) [inline front/src/components/Chat/clickableMenu.tsx:28]: This seems useless for now :grin:

**Review by KobeLaC — APPROVED** [review]
Nice !

File: front/src/components/Chat/clickableMenu.tsx (hunk 1 above)
Comment (se-lee) [reply front/src/components/Chat/clickableMenu.tsx:28]: ah I know 
i did this in advance before i forget lol
guillaume will do the back very soon so i'll just leave it like this and uncomment it once the back is ready

**Review by emmanuel-lbs — APPROVED** [review]

**Pull request merged**